iothub/device/src/Transport/AmqpIoT/AmqpIoTSession.cs

```diff
@@ -94,9 +98,9 @@ TimeSpan timeout
         {
             return await OpenReceivingAmqpLinkAsync(
                 deviceIdentity,
-                AmqpSession,
+                _amqpSession,
                 null,
-                (byte)ReceiverSettleMode.First,
+                (byte)ReceiverSettleMode.Second,
```

Comment:
Settle mode Second is only valid for C2D where the application must call the disposition API.
For everything else it should be First as it is in previous versions of the AMQP stack.

Reply:
This is C2D here right?